### PR TITLE
`RadarMap.fitToMarkers` class method

### PR DESCRIPTION
**Pull request by jaspk06**
NOTE: https://github.com/radarlabs/radar-sdk-js/pull/153 needs to be reviewed first

single line function to adjust the bounds of a `RadarMap` to fit to all `RadarMarker` added to the map.

### CODE REVIEW

File: src/ui/RadarMap.ts [hunk 1]
```diff
@@ -155,6 +155,15 @@ class RadarMap extends maplibregl.Map {
     }
     this.on('styledata', onStyleLoad);
   }
+
+  fitToMarkers(fitBoundsOptions: maplibregl.FitBoundsOptions = { padding: 50 }, overrideMarkers?: RadarMarker[]) {
+    const bounds = new maplibregl.LngLatBounds();
+    const markers = overrideMarkers || this._markers;
+    markers.forEach(marker => {
```
Comment (kochis) [inline src/ui/RadarMap.ts:162]: We should probably add a lint rule for this, but there should be parens around the arguments
```
markers.forEach((markers) => {
```

File: src/ui/RadarMap.ts [hunk 2]
```diff
@@ -155,6 +155,15 @@ class RadarMap extends maplibregl.Map {
...
+      bounds.extend(marker.getLngLat());
+    });
+    this.fitBounds(bounds, fitBoundsOptions);
```
Comment (kochis) [inline src/ui/RadarMap.ts:165]: What does this do if markers is empty?

Comment (jaspk06) [reply src/ui/RadarMap.ts:150]: added a no-op if markers is empty. Yeah you can pass in animation options into `fitBoundsOptions` https://docs.mapbox.com/mapbox-gl-js/api/map/#map#fitbounds

Comment (jaspk06) [reply src/ui/RadarMap.ts:150]: just do `fitMarkers({animate: false})` to disable animations. it uses `map.flyTo` by default

File: src/ui/RadarMap.ts [hunk 3]
```diff
@@ -135,6 +135,20 @@ class RadarMap extends maplibregl.Map {
   getMarkers(): RadarMarker[] {
     return this._markers;
   }
+
+  fitToMarkers(fitBoundsOptions: maplibregl.FitBoundsOptions = { padding: 50 }, overrideMarkers?: RadarMarker[]) {
+    const bounds = new maplibregl.LngLatBounds();
```
Comment (kochis) [inline src/ui/RadarMap.ts:140]: nit: move this down to just before the forEach. No need to initialize if we're going to return early.

**Review by kochis — APPROVED** [review]

**Pull request merged**